replica/table.cc

```diff
@@ -2040,6 +2041,24 @@ future<db::replay_position> table::discard_sstables(db_clock::time_point truncat
     co_return p->rp;
 }
 
+void table::mark_ready_for_writes(db::commitlog* cl) {
+    update_sstables_known_generation(sstables::generation_from_value(0));
+    if (_config.enable_commitlog) {
+        _commitlog = cl;
+    }
+    _readonly = false;
+}
+
+db::commitlog* table::commitlog() const {
+    if (_virtual_writer) {
+        return nullptr;
+    }
+    if (_readonly) {
```

Comment:
Let's add ` [[unlikely]]` to both `if`s.

Reply:
done.

BTW, do you know the practical consequences of this? Does it really help with mispredictions?